account_multicurrency_revaluation_type/wizard/wizard_currency_revaluation_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_config_settings" model="ir.ui.view">
```

Comment:
`view_account_currency_revaluation_wizard` (`view_account_config_settings` already used in `views/res_config_view.xml`)